.github/workflows/run_pytest.yaml

```diff
@@ -12,16 +12,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
           - uses: actions/checkout@v4
-          
+
```

Comment:
Are these newlines intentional?

Reply:
I recreated this PR in https://github.com/civiform/cloud-deploy-infra/pull/390 from a branch of this repo instead of my own repo.

I'll fix this in that PR.

Reply:
Hmm actually the newline at the end of the file is added by vim automatically per the POSIX standard: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206